examples/lb3models/README.md

```diff
@@ -0,0 +1,94 @@
+# @loopback/example-lb3models
+
+This project demonstrates how to add LoopBack 3 models into a LoopBack 4
+application.
+
+## Instructions
+
+1. Create a new LB4 app using `lb4 app`. Add `@loopback/v3compat'` to project
+   dependencies.
+
+2. Copy your model files from `common/models` to `legacy/models` directory, for
+   example:
+
+   ```text
+   legacy/models/coffee-shop.js
+   legacy/models/coffee-shop.json
+   ```
+
+   IMPORTANT! These files must live outside your `src` directory, out of sight
+   of TypeScript compiler.
+
+3. Copy your `server/model-config` to `src/legacy/model-config.json`.
+
+   Remove references to LoopBack 3 built-in models like User and ACL. LoopBack 4
+   does not support local authentication yet.
+
+   Remove `_meta` section, LoopBack 4 does not support this config option.
+
+4. Modify your Application class and apply `CompatMixin`.
+
+   ```ts
+   import {CompatMixin} from '@loopback/v3compat';
+   // ...
+
+   export class TodoListApplication extends CompatMixin(
+     BootMixin(ServiceMixin(RepositoryMixin(RestApplication))),
+   ) {
+     // ...
+   }
+   ```
+
+5. Edit your boot configuration in `src/application.ts` and add the following
+   section:
+
+   ```ts
+   {
+     v3compat: {
+        // from "/dist/src/application.ts" to "/legacy"
+       root: '../../legacy',
+     },
+   }
+   ```
+
+6. Register your legacy datasources in Application's constructor.
```

Comment:
To clarify, at this point, our compat lb3-model booter would have configured and loaded the legacy models for us right? How do we attach legacy datasource(s) to a legacy model after?